week02/day10/README.md

```diff
@@ -226,4 +226,4 @@ Only one feature to go. That's tomorrow's topic. And you'll learn more about def
 * Microsoft, [Operator Overloading (F#)](https://msdn.microsoft.com/en-us/library/dd233204.aspx)
 * Wikibooks, [F Sharp Programming / Operator Overloading](https://en.wikibooks.org/wiki/F_Sharp_Programming/Operator_Overloading)
 
-* Scott Wlaschin, [Understanding type inference](http://fsharpforfunandprofit.com/posts/type-inference/)
\ No newline at end of file
+* Scott Wlaschin, [Understanding type inference](http://fsharpforfunandprofit.com/posts/type-inference/)
```

Comment:
Apparently Git shows this line as changed, though no changes have been made to that one.